us/en/skymiles/blocks/toc/toc.js

```diff
@@ -0,0 +1,27 @@
+export default function decorate(block) {
+  const currentToc = block.querySelector('ul');
+
+  if (currentToc === null) {
```

Comment:
Do we have an `else` for this? where you'd essentially take whatever the author has put in there, or you leave the markup untouched?

I'd also suggest swapping out for a guard clause instead so you reduce indentation:
```js
if (!!currentToc) {
  return;
}

…
```

Reply:
Yes. I am leaving the markup untouched here for the third scenario:
`ToC block with authored content (i.e. the table heading + 1 cell with a list of links) => take the list of anchors+bookmarks and just style it with CSS`

Reply:
Addition of guard clause. (Done)